basics/data-import/pinot-stream-ingestion/README.md

```diff
@@ -77,40 +78,31 @@ For our sample data, the schema configuration looks like this:
+| `stream.[streamType].decoder.class.name`              | Name of class to parse the data. The class should implement the `org.apache.pinot.spi.stream.StreamMessageDecoder` interface.                                                         | String. Available options:  - `org.apache.pinot.plugin.inputformat.json.JSONMessageDecoder`  - `org.apache.pinot.plugin.inputformat.avro.KafkaAvroMessageDecoder` - `org.apache.pinot.plugin.inputformat.avro.SimpleAvroMessageDecoder` - `org.apache.pinot.plugin.inputformat.avro.confluent.KafkaConfluentSchemaRegistryAvroMessageDecoder`  - `org.apache.pinot.plugin.inputformat.csv.CSVMessageDecoder`  - `org.apache.pinot.plugin.inputformat.protobuf.ProtoBufMessageDecoder` |
+| `stream.[streamType].consumer.factory.class.name`     | Name of factory class to provide the appropriate implementation of low-level and high-level consumer, as well as the metadata                                                         | String. Available options:  - `org.apache.pinot.plugin.stream.kafka09.KafkaConsumerFactory` - `org.apache.pinot.plugin.stream.kafka20.KafkaConsumerFactory` - `org.apache.pinot.plugin.stream.kinesis.KinesisConsumerFactory` - `org.apache.pinot.plugin.stream.pulsar.PulsarConsumerFactory`                                                                                                                                                                                         |
+| `stream.[streamType].consumer.prop.auto.offset.reset` | Determines the offset from which to start the ingestion                                                                                                                               | - `smallest` - `largest` - timestamp in milliseconds                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `stream.[streamType].decoder.prop.format`             |                                                                                                                                                                                       | - `JSON`                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Need a definition for this one

Reply:
for now, we could say something like:
Specifies the data format to ingest via a stream. The value of this property should match the format of the data in the stream.